modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -611,6 +634,82 @@ public ObjectEntry updateObjectEntry(
 					dtoConverterContext.getUserId())));
 	}
 
+	private void _addNestedObjectEntries(
+			long groupId, DTOConverterContext dtoConverterContext,
+			ObjectDefinition objectDefinition,
+			com.liferay.object.model.ObjectEntry objectEntryModel,
+			ObjectEntry objectEntry)
+		throws Exception {
+
+		ObjectRelationship objectRelationship = null;
+
+		Map<String, Object> objectEntryProperties = objectEntry.getProperties();
+
+		for (Map.Entry<String, Object> property :
+				objectEntryProperties.entrySet()) {
+
+			objectRelationship =
+				_objectRelationshipLocalService.
+					fetchObjectRelationshipByObjectDefinitionId(
+						objectDefinition.getObjectDefinitionId(),
+						property.getKey());
+
+			if ((objectRelationship == null) &&
+				(property.getValue() instanceof Map)) {
+
+				objectRelationship = _getRelatedObjectRelationship(
+					objectDefinition, property);
+
+				if (objectRelationship == null) {
```

Comment:
Should't this be a `continue`? With `break` you jump out of the whole loop, that means the rest of the fields won't be processed.

Reply:
I agree with you about to change the `break` to `continue`. The problem is that we could iterate into some properties and we have to check all them just to make sure that a relationship name is used, if not jump it.